### DISCUSSION TITLE
can't find docker image when deploy 2.0.0-alpha docker

### DISCUSSION THREAD

**Discussion by shuitao** [apache/dolphinscheduler#6631] [Q&A]
use docker-compose up, 
manifest for apache/dolphinscheduler:2.0.0-snapshot nor found:manifest unknown

**Comment by zhongjiajie** [comment]
Hello @shuitao please use command `docker pull apache/dolphinscheduler:2.0.0-alpha` to get `2.0.0-alpha` images

**Reply by zhongjiajie** [reply]
BTW, please use english in discussion

**Comment by shuitao** [comment]
sorry for that，will use eng。

**Reply by zhongjiajie** [reply]
And did you solve this issue now? @shuitao

**Reply by shuitao** [reply]
problem solved
deploy successfully，
login with admin---ok
create project--fail， no extra info

**Reply by zhongjiajie** [reply]
> create project--fail， no extra info

Could you go and see if docker log have some failure information?
